### PR TITLE
Adding missing team members

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

File: pages/team.html [hunk 1]
```diff
@@ -32,16 +32,44 @@ <h2> Team Members</h2>
...
+
                 {% if teamMember.role != nil %}
-                  <p class="member-description"><b>Role:</b> {{ teamMember.role }}</p>
+                  <p class="member-description"> My role is <strong>{{ teamMember.role }}.</strong> </p>
+                {% endif %}
+                {% if teamMember.thinking != nil %}
+                  <p> 🤔 I am thinking about: <strong>{{ teamMember.thinking }}.</strong> </p>
+                {% endif %}
+                {% if teamMember.doing != nil %}
+                  <p> I am doing: <strong>{{ teamMember.doing }}.</strong> </p>
```
Comment (JacOng17) [inline pages/team.html]:
```suggestion
                  <p> Things I know & do: <strong>{{ teamMember.doing }}.</strong> </p>
```

**Comment by CharlotteHinton** [comment]
@Br3nda 
- [ ] Hamish title - Principal Advisor Better Rules
- [ ] Hazel's title - Principal Advisor Emerging Technologies
- [ ] Darryl doesn't have a role?
- [ ] Felipe - remove the full stop after the ? on 'what am I thinking about'
- [ ] Grant doesn't have a role?
- [ ] Meredith doesn't have a role?
- [ ] Rob doesn't have a role?
- [ ] Tim's title is wrong
- [ ] Designer should be capitalised for Karyn and Ross

- [ ] Remove the .. after each sentence

**Comment by Driedfrog** [comment]
My Title: Is Emerging Technologies Lead

**Comment by Driedfrog** [comment]
Hazel Title:
 Emerging Technologies Lead

**Comment by robholmesnz** [comment]
My role is Senior Product Owner

"in-equality" --- needs to be updated to --> "inequality"

(the rest is fine).

File: _authors/hazel-bradshaw.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: hazel-bradshaw
+first_name: Hazel
+last_name: Bradshaw
+full_name: Hazel Bradshaw
+role: Principal Advisor Emerging Technologies
+image: "/assets/img/team/hazel.jpg"
+active: true
+github: Driedfrog
+thinking: Emerging tech. Game design
```
Comment (JacOng17) [inline _authors/hazel-bradshaw.md]:
```suggestion
thinking: Emerging technologies and environmental-centred design
```

File: _authors/hazel-bradshaw.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+doing: Human Interface design. Creative technology. Game design & Psychology. Design strategy & innovation. Virtual Reality. Research & Measurements. Prototyping.
```
Comment (JacOng17) [inline _authors/hazel-bradshaw.md]:
```suggestion
doing: Emerging technologies disruption and impact, digital transformation, human interface technology, game design psychology, rapid prototyping and tech-futures.
```

File: _authors/hazel-bradshaw.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+name: hazel-bradshaw
+first_name: Hazel
+last_name: Bradshaw
+full_name: Hazel Bradshaw
+role: Principal Advisor Emerging Technologies
```
Comment (JacOng17) [inline _authors/hazel-bradshaw.md]:
```suggestion
role: Emerging Technologies Lead
```

File: _authors/jacob-ong.md [hunk 5]
```diff
@@ -3,7 +3,9 @@ name: jacob-ong
 first_name: Jacob
 last_name: Ong
 full_name: Jacob Ong
-role: Developer
+role: Software Engineer
 image: "/assets/img/team/jacob-square.jpg"
 active: true
+thinking: Personal growth
+doing: Gym, soccer, computers
```
Comment (JacOng17) [inline _authors/jacob-ong.md]:
```suggestion
doing: Gym, soccer, computers
github: JacOng17
```

File: _authors/dave-moskovitz.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+name: dave-moskovitz
+first_name: Dave
+last_name: Moskovitz
+full_name: Dave Moskovitz
```
Comment (Br3nda) [inline _authors/dave-moskovitz.md]: github
```suggestion
full_name: Dave Moskovitz
github: davemosk
```

File: _authors/hamish-fraser.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+name: hamish-fraser
+first_name: Hamish
+last_name: Fraser
+full_name: Hamish Fraser
+role: Principal Advisor Better Rules
```
Comment (Br3nda) [inline _authors/hamish-fraser.md]:
```suggestion
role: Principal Advisor Better Rules
github: verbman
```

File: _authors/merridy-marshall.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+name: merridy-
```
Comment (Br3nda) [inline _authors/merridy-marshall.md]:
```suggestion
name: merridy-marshall
```

File: pages/team.html [hunk 9]
```diff
@@ -32,16 +32,49 @@ <h2> Team Members</h2>
...
                     {% endif %}
                   </div>
-                <h3 class="member-tag"><b><a href="/author/{{ teamMember.name }}">{{ teamMember.full_name }}</a></b></h3>
+                <h3 class="member-tag">{{ teamMember.full_name }}</h3>
                 {% if teamMember.role != nil %}
-                  <p class="member-description"><b>Role:</b> {{ teamMember.role }}</p>
+                  <p class="member-description"> My role is <strong>{{ teamMember.role }}.</strong> </p>
                 {% endif %}
+                {% if teamMember.thinking != nil %}
+                  <p> 🤔 I am thinking about: <strong>{{ teamMember.thinking }}</strong> </p>
```
Comment (Br3nda) [inline pages/team.html]:
```suggestion
                  <p>🤔 I am thinking about: <strong>{{ teamMember.thinking }}</strong> </p>
```

File: _authors/glen-thurston.md [hunk 10]
```diff
@@ -6,5 +6,8 @@ full_name: Glen Thurston
 role: Design Lead
 image: "/assets/img/team/glen-square.jpg"
 active: true
+github: gthurstonnz
 published: true
+thinking: Developing out team for this year.
```
Comment (gthurstonnz) [inline _authors/glen-thurston.md]: Need to change the 'out' to 'our'

File: _authors/charlotte-hinton.md [hunk 11]
```diff
@@ -4,6 +4,9 @@ first_name: Charlotte
 last_name: Hinton
 full_name: Charlotte Hinton
 image: "/assets/img/team/charlotte-square.jpg"
-active: true
+active: false
+github: CharlotteHinton
 published: true
+thinking: All the things
+doing: Kanban, Lean, Getting things done, coaching, Agile Welly, NZ Birds, ACtive Listning Nurturing Teams, Sour dough, Collatoration, Mentoring
```
Comment (Felipelab) [inline _authors/charlotte-hinton.md]: ACtive Listning > Active Listening

File: _authors/felipe-bohorquez-perry.md [hunk 12]
```diff
@@ -6,4 +6,6 @@ full_name: Felipe Bohorquez-Perry
 role: Scrum Master
 image: "/assets/img/team/felipe-square.jpg"
 active: true
+thinking: Where is everything?
```
Comment (Felipelab) [inline _authors/felipe-bohorquez-perry.md]: thinking: Everything I can take in, in English, and, occasionally in Spanish

File: _authors/felipe-bohorquez-perry.md [hunk 13]
```diff
@@ -6,4 +6,6 @@ full_name: Felipe Bohorquez-Perry
...
+doing: AGILE. Design thinking. Working with devs. Scrum, Film, VFX, software, learning.
```
Comment (Felipelab) [inline _authors/felipe-bohorquez-perry.md]: Agile, Scrum, pragmatism, value with values, better ways of working, new tools, old tools, human behaviours, incentivizing our best impulses, Music, VFX, films and how it all connects.

**Comment by Br3nda** [comment]
I've got most of the missing alumni added. I'm going to leave active team members to add their own missing details. (there's instructions on this Repo)

**Pull request merged**